get_all_manifests.sh

```diff
@@ -32,16 +59,20 @@ cp -r ./.odh-manifests-tmp/kserve/ ./odh-manifests
 ls -lat ./odh-manifests/
```

Comment:
wanna remove this one? thought i cleaned it up, but looks like it is still there :/

Reply:
```suggestion
```